### PR TITLE
add amg and mix

### PR DESCRIPTION
**Pull request by shengwenzheng**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/rangers-dehero exports TVL: 
        
 
 
    rpg                       22.69 M
    
    total                    22.69 M

**Comment by shengwenzheng** [comment]
add amg and mix

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
are you trying to count amg and mix supply as tvl? you need to give more details as to why this should be treated as tvl

**Comment by shengwenzheng** [comment]
hello:
$AMG: $AMG is the governance token of DeHeroGame, an NFT TCG pioneering GameFi 2.0, and has a total supply of 100,000,000. $AMG demonstrates solid value growth by representing the prime glory and governing power among the game community. The diverse usage scenarios of $AMG include in-game NFT progression, monthly membership subscription, special event voucher, voting, and more. $AMG will not only be available on popular exchanges but be rewarded to advanced and loyal DeHeroGame players as well.

$MIX:
MlX is a platform token representing the growth of the platform's ecosystem value, witha fixed circulation of 10 billion. lts distribution rules are based on community members behaviors beneficial to the ecosystem, such as marketing, developing applications, creating content, and providing services, which are all rewarded with MlX.

MlX holders can enjoy distinguished rights and interests:
- Management rights and interests: participate in voting various decisions on the platformand the governance of the platform's ecosystem construction.
- Privileged rights and interests: according to the MlX holdings, MlX holders have privileged rights and interests to enjoy various services on the platform.
- Airdrop benefits: new games released on the MixMarvel platform will be recommended to hold airdrops to MIX community users. MlX community users will have the opportunity to learn about the most recent games first and participate in advance.

In summary, both tokens are important for users within the ecosystem, and users can use both tokens to accomplish a variety of tasks and transactions.



At 2024-04-24 15:02:21, "g1nt0ki" ***@***.***> wrote:

@g1nt0ki requested changes on this pull request.

are you trying to count amg and mix supply as tvl? you need to give more details as to why this should be treated as tvl

**Comment by shengwenzheng** [comment]
hello
Full details about MIX and AMG have been sent to you, do you have any other questions?



At 2024-04-24 15:02:21, "g1nt0ki" ***@***.***> wrote:

@g1nt0ki requested changes on this pull request.

are you trying to count amg and mix supply as tvl? you need to give more details as to why this should be treated as tvl

**Comment by g1nt0ki** [comment]
based on the info you provided, neither should be counted as tvl: https://docs.llama.fi/list-your-project/what-to-include-as-tvl
